packages/edit-site/src/components/header/document-actions/style.scss

```diff
@@ -34,20 +34,19 @@
 	.edit-site-document-actions__secondary-item {
 		font-size: $default-font-size;
 		opacity: 0;
-		transform: translateY(20px);
+		transform: translateY(0);
 		transition: transform 0.2s;
 		@include reduce-motion(transition);
 	}
 
 	&.has-secondary-label {
-		.edit-site-document-actions__title-wrapper {
-			transform: translateY(-12px);
-		}
+		transform: translateY(10px);
 
 		.edit-site-document-actions__title {
-			position: absolute;
 			color: $gray-900;
 			font-weight: bold;
+			white-space: nowrap;
```

Comment:
I think we should add this to `.edit-site-document-actions__title` even when secondary label is not visible and make the styling consistent with this. If we allow wrapping in that case it will wrap very long titles, and then convert into a one line after animating, which makes the animation a bit jarring imo.